src/js/controller.js

```diff
@@ -448,7 +451,7 @@ export const MediaUIStates = {
   MEDIA_SUBTITLES_LIST: {
     get: function (controller) {
       // TODO: Move to non attr specific values
-      return stringifyTextTrackList(getSubtitleTracks(controller)) || undefined;
+      return getSubtitleTracks(controller).map(({ kind, label, language }) => ({ kind, label, language })) || undefined;
```

Comment:
Lets return an empty array here, or null. Or help me understand why returning `undefined` is correct. I'm not sure why we'd ever do that over something more distinguishable from "property does not exist".